main.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
Let's make this `cmd/finder/main.go` or something similar.

Reply:
Maybe add a `// import "github.com/gofrs/help-requests/cmd/finder"` comment here too?

Perhaps a package-level comment would be nice too, like here https://github.com/golang/tools/blob/master/cmd/stringer/stringer.go